code/src/wattrex_battery_cycler/mid/mid_data/mid_data_battery.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+'''
+Definition of MID DATA battery related info.
+'''
+#######################        MANDATORY IMPORTS         #######################
+from __future__ import annotations
+
+#######################         GENERIC IMPORTS          #######################
+
+#######################       THIRD PARTY IMPORTS        #######################
+
+#######################      SYSTEM ABSTRACTION IMPORTS  #######################
+from system_logger_tool import sys_log_logger_get_module_logger
+log = sys_log_logger_get_module_logger(__name__)
+
+#######################          PROJECT IMPORTS         #######################
+
+#######################          MODULE IMPORTS          #######################
+from .mid_data_experiment import MidDataPwrRangeC
+#######################              ENUMS               #######################
+
+
+#######################             CLASSES              #######################
+
+class MidDataBatteryC:
+    '''
+    Generic battery info.
+    '''
+    # pylint: disable=too-many-arguments
+    def __init__(self, name: str, model : str, volt_min : int = 800,
```

Comment:
All params must be initialized to None

Reply:
Done, also Lithium Bat and Redox Bat initialize to None

Reply:
To 0

Reply:
Why these numbers are used?